ci/azure-pipelines-nightly.yml

```diff
@@ -56,6 +56,14 @@ jobs:
   steps:
   - template: ../ci/checkout-template.yml
   - template: ../ci/install-package-dependencies.yml
+  - bash: |
+      ci/bazelisk.sh run \
```

Comment:
You should explain why we need to do it twice. That's because (for reasons not totally understood), rusb cannot dynamically update its device list in CI (udev is not completely functional). If the device is in normal mode, the first thing that opentitantool does is to switch it to DFU mode and wait until it reconnects. This reconnection is never detected. But if we run the tool another time, the device list is queried again and opentitantool can finish the update. The device will now reboot in normal mode and work for the hyperdebug job.

Reply:
Done